index.html

```diff
@@ -0,0 +1,219 @@
+                        <li>2015</li>
+                      </ul>
+                    </div>
+          
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="prog-lang">
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javaScript</li>
+                    </ul>
+                    <a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a>
+                  </div>
+            </div>
+            <section class="about-myself">
+
+                <section>
+                    <h1>About <br>Myself</h1>
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <a>LET'S CONNECT</a>
+                <ul class="socialnetworks">
+                <li>
+                    <a href="https://twitter.com/home">
+                        <img src="twit.svg" alt="Twiter">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/feed/">
+                        <img src="ln.svg" alt="Linkedin">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://medium.com/">
+                        <img src="m.svg" alt="Medium">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://github.com/djo1975">
+                        <img src="cat.svg" alt="Github">
+                    </a>
+                </li>
+                <li>
+                   <a href="https://github.com/djo1975">
+                        <img src="v.svg" alt="Victory">
+                    </a>
+                </li>
+                </ul>
+                </div>
+                <div class="flex-item">
+                  <a href="" class="button resume-btn-border">Get my resume</a>
+                </div>
+        
+                <div class="flex-item">
+                  <ul class="rest">
+                      <li class="arrow"><img src="jsc.svg" alt="Down Arrow"></li>
+                    </ul>
+                    <section class="languages">
+                      <li><img src="js.svg"> JavaScript</li>
+                      <li><img src="html.svg"> HTML </li>
+                      <li><img src="css.svg">  CSS </li>
+                    </section>
+        <section class="other">
+              <li class="other-item">Frameworks</li>
+              <li class="others-item-2"><img src="jsc.svg" alt="Right Arrow"></li>
+            </section>
+            <section class="others-section-ii">
+              <li class="others-item-3">Skills</li>
+              <li class="others-arrow"><img src="jsc.svg" alt="Right Arrow"></li>
+            </section>

```

Comment:
- Good job on the work done so far, but there are still some differences that need to be addressed. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
-- This section has no background image, but you used added some background images on your page, please correct this 👍 👍 
--The `Get my Resume` button doesn't match the design in the Figma template, please fix this by giving it some more width 🙏 🙏 
-- The following sections are missing in your design, I have highlighted them in the Figma template attached below
    --- The list of technologies under the languages section, please fix this 🙏 
<img width="232" alt="rdbfb" src="https://user-images.githubusercontent.com/73607512/205453371-3aba49f9-6509-4c37-a4f4-cde79f677b60.PNG">

Figma Design:
-- Please pay attention to all marked sections and include them in your design
<img width="170" alt="utukk" src="https://user-images.githubusercontent.com/73607512/205453382-b5595432-8c47-49bb-a0ca-703fbbe44fe7.PNG">